sharding/syncer/service_test.go

```diff
@@ -32,24 +37,27 @@ func TestStartStop(t *testing.T) {
 
 	syncer, err := NewSyncer(params.DefaultConfig, &mainchain.SMCClient{}, server, shardChainDB, shardID)
 	if err != nil {
-		t.Fatalf("Unable to setup syncer service: %v", err)
+		t.Fatalf("Unable to setup sync service: %v", err)
 	}
 
 	syncer.Start()
+
 	h.VerifyLogMsg("Starting sync service")
 
 	if err := syncer.Stop(); err != nil {
-		t.Fatalf("Unable to stop syncer service: %v", err)
+		t.Fatalf("Unable to stop sync service: %v", err)
 	}
+
 	h.VerifyLogMsg("Stopping sync service")
 
-	// The context should have been cancelled.
+	// The context should have been canceled.
 	if syncer.ctx.Err() == nil {
-		t.Error("Context was not cancelled")
+		t.Error("Context was not canceled")
```

Comment:
Interestingly, both spellings are correct, but this is the American spelling so I suppose it is more correct 🇺🇸 

https://www.grammarly.com/blog/canceled-vs-cancelled/

Reply:
Loll haha let's stick to that!